dask/compatibility.py

```diff
@@ -73,6 +78,16 @@ def _getargspec(func):
     operator_div = operator.div
     FileNotFoundError = IOError
 
+    def _make_reraise():
+        _code = ("def reraise(exc, tb=None):"
+                "    raise type(exc), exc, tb")
+        namespace = {}
+        exec("exec _code in namespace")
+        return namespace['reraise']
+
+    reraise = _make_reraise()
+    del _make_reraise
```

Comment:
Was trying to minimize extraneous things in the namespace, but that's probably unnecessary.